examples/containerservice/v1beta2/kubernetescluster.yaml

```
@@ -60,3 +62,16 @@ metadata:
 spec:
   forProvider:
     location: West Europe
+
+---
+
+apiVersion: managedidentity.azure.upbound.io/v1beta1
+kind: UserAssignedIdentity
+metadata:
+  labels:
+    testing.upbound.io/example-name: exmple-managedidentity
+  name: example-managedidentity
+spec:
+  forProvider:
+    location: West Europe
+    name: exmple-managidentity
```

We should add `resourceGroupName` because it is a required field.

```suggestion
    name: exmple-managidentity
    resourceGroupNameSelector:
      matchLabels:
        testing.upbound.io/example-name: example-containerservice
```